apistructs/iteration.go

```diff
@@ -147,6 +147,10 @@ type ISummary struct {
 }
 
 type ISummaryState struct {
-	Done   int `json:"done"`
-	UnDone int `json:"undone"`
+	Done        int       `json:"done"`
+	UnDone      int       `json:"undone"`
+	Total       int       `json:"total,omitempty"`
+	IssueType   IssueType `json:"issue_type,omitempty"`
+	State       int64     `json:"state,omitempty"`
+	IterationID int64     `json:"iteration_id,omitempty"`
```

Comment:
does necessary to export these fields to api?

Reply:
you need to define a new model for dao.query only, not share the struct with api

Reply:
adjusted